docs/DeveloperGuide.md

```diff
@@ -2,36 +2,118 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}  
+Zhou Zhou's Individual Project: https://github.com/maanyos/ip \
+Nay Chi's Individual Project: https://github.com/NayChi-7/ip
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Class Diagrams
+
+Biometrics component:  
```

Comment:
for each component/section, maybe you can bold/underline the title so that each section is more visible and can be differentiated from the other sections